prowler/providers/aws/services/elasticache/elasticache_redis_cluster_automatic_failover_enabled/elasticache_redis_cluster_automatic_failover_enabled.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticache_redis_cluster_automatic_failover_enabled",
+  "CheckTitle": "Ensure Elasticache Redis clusters have automatic failover enabled.",
+  "CheckType": [
+    "AWS Foundational Security Best Practices"
```

Comment:
Please, use the check types from https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#Types